docker/compose.go

```diff
@@ -15,14 +16,25 @@ const (
 	stopTimeoutSecs = 2
 )
 
+func composeCommand() string {
```

Comment:
I feel like we should cache the result vs looking it up every time so a single run is consistent (and doesn't produce these logs multiple times)

Reply:
Fixed in a190a16